spec/CloudCode.spec.js

```diff
@@ -50,6 +50,97 @@ describe('Cloud Code', () => {
     });
   });
 
+  it('can get config', () => {
+    const config = Parse.Server;
+    let currentConfig = Config.get('test');
+    expect(Object.keys(config)).toEqual(Object.keys(currentConfig));
+    expect(Object.keys(config).sort()).toEqual(
```

Comment:
I'm not sure about this list here; it will need to be maintained with every change. It should be enough to test deep quality in `expect(Object.keys(config)).toEqual(Object.keys(currentConfig));`